aten/src/ATen/native/native_functions.yaml

```diff
@@ -1741,6 +1741,15 @@
   python_default_init:
     p: 2
 
+- func: frobenius_norm(Tensor self) -> Tensor
+  variants: method, function
```

Comment:
`variants` are `method` and `function` by default. I don't think it's necessary to specify them explicitly.